server/website/website/views.py

```diff
@@ -1429,29 +1432,36 @@ def create_test_website(request):  # pylint: disable=unused-argument
                                          password='ottertune_test_user')
     test_project = Project.objects.create(user=test_user, name='ottertune_test_project',
                                           creation_time=now(), last_update=now())
+    hyper_parameters = JSONUtil.dumps(utils.get_constants())
+
     # create no tuning session
     s1 = Session.objects.create(name='test_session_no_tuning', tuning_session='no_tuning_session',
                                 dbms_id=1, hardware=test_hardware, project=test_project,
                                 creation_time=now(), last_update=now(), user=test_user,
-                                upload_code='ottertuneTestNoTuning')
+                                upload_code='ottertuneTestNoTuning',
+                                hyper_parameters=hyper_parameters)
```

Comment:
Once you update the column default value to the json string of constants.py you don't need to pass it in to `create`.